ci/image/Dockerfile.nodejs-centos7-node12-test

```diff
@@ -7,7 +7,7 @@ RUN yum -y install centos-release-scl
 
 # nvm environment variables
 ENV NVM_DIR /usr/local/nvm
-ENV NODE_VERSION 12.0.0
+ENV NODE_VERSION 14.17.0
```

Comment:
Should this file (and docker image) still be named with node12 inside? Now it will be running on node 14?

Reply:
prefarably we need to change the docker image name. but i can build and confirm if this is working though.

Reply:
Do you have to have it merged to build image? Maybe we shouldn't change this image in this PR?